docs/pages/installation/platforms.md

```diff
@@ -7,7 +7,7 @@ permalink: installation/platforms
 The table below provides installation instructions for each platform that Meshery is compatible with:
 
 | Platform      | Version       |
-| -------------:|:-------------|   
+|  &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;||   
```

Comment:
How do we clean up this table, but leave the visually appealing alternating row color?

Reply:
Another way to adjust the spacing to make an HTML Table and then adjust it. I thought this could more of a workaround rather than re-writing the complete table.

Reply:
Oops. I have commented on the wrong line.

Rather, what I mean is that the existing, alternating table row colors like this are nice:

<img width="524" alt="Screen Shot 2020-05-20 at 6 33 43 AM" src="https://user-images.githubusercontent.com/7570704/82441708-f325c500-9a63-11ea-88e9-fe9c15d83a29.png">

as versus without the row colors:

<img width="611" alt="Screen Shot 2020-05-20 at 6 36 33 AM" src="https://user-images.githubusercontent.com/7570704/82441994-734c2a80-9a64-11ea-8ea3-3a78257373bc.png">

Reply:
I will see into it, I was getting shown the design initially.

Reply:
Got it! will be pushing fix with the next commit.

Reply:
Fixed!